android/src/main/kotlin/com/homex/open_mail_app/OpenMailAppPlugin.kt

```diff
@@ -115,6 +130,58 @@ class OpenMailAppPlugin : FlutterPlugin, MethodCallHandler {
         }
     }
 
+    private fun composeNewEmailAppIntent(@NonNull chooserTitle: String, @NonNull contentJson: String): Boolean {
+        val packageManager = applicationContext.packageManager
+        val emailContent = Gson().fromJson(contentJson, EmailContent::class.java)
+        val emailIntent = Intent(Intent.ACTION_SENDTO, Uri.parse("mailto:"))
+
+        val activitiesHandlingEmails = packageManager.queryIntentActivities(emailIntent, 0)
+        if (activitiesHandlingEmails.isNotEmpty()) {
+            val emailAppChooserIntent = Intent.createChooser(Intent(Intent.ACTION_SENDTO).apply {
+                data = Uri.parse("mailto:")
+                type = "text/plain"
+                setClassName(activitiesHandlingEmails.first().activityInfo.packageName, activitiesHandlingEmails.first().activityInfo.name)
+
+                putExtra(Intent.EXTRA_EMAIL, emailContent.to.toTypedArray())
+                putExtra(Intent.EXTRA_CC, emailContent.cc.toTypedArray())
+                putExtra(Intent.EXTRA_BCC, emailContent.bcc.toTypedArray())
+                putExtra(Intent.EXTRA_SUBJECT, emailContent.subject)
+                putExtra(Intent.EXTRA_TEXT, emailContent.body)
+            }, chooserTitle)
+
+            val emailComposingIntents = mutableListOf<LabeledIntent>()
+            for (i in 1 until activitiesHandlingEmails.size) {
+                val activityHandlingEmail = activitiesHandlingEmails[i]
+                val packageName = activityHandlingEmail.activityInfo.packageName
+                    emailComposingIntents.add(
+                        LabeledIntent(
+                                Intent(Intent.ACTION_SENDTO).apply {
+                                    data = Uri.parse("mailto:")
+                                    type = "text/plain"
+                                    setClassName(activityHandlingEmail.activityInfo.packageName, activityHandlingEmail.activityInfo.name)
+                                    putExtra(Intent.EXTRA_EMAIL, emailContent.to.toTypedArray())
+                                    putExtra(Intent.EXTRA_CC, emailContent.cc.toTypedArray())
+                                    putExtra(Intent.EXTRA_BCC, emailContent.bcc.toTypedArray())
+                                    putExtra(Intent.EXTRA_SUBJECT, emailContent.subject)
+                                    putExtra(Intent.EXTRA_TEXT, emailContent.body)
+                                },
+                            packageName, 
```

Comment:
nit: According to `ktlint`, no trailing space
```suggestion
                            packageName,
```